Source/IO/BasicROOTFileWriter/KTBasicROOTTypeWriterSpectrumAnalysis.cc

```diff
@@ -955,9 +962,13 @@ namespace Katydid
         unsigned nComponents = sumData.GetNComponents();
 
         if (! fWriter->OpenAndVerifyFile()) return;
+        unsigned optimizedGridIndex=sumData.GetOptimizedGridPoint();
 
         for (unsigned iChannel=0; iChannel<nComponents; iChannel++)
         {
+            std::cout<< "::iChannel << <<optimizedGridIndex"<<std::endl;
```

Comment:
KTLogger

Reply:
Hi @nsoblath  , the cout statements were not needed. I forgot to remove it. They are now removed.